### PR TITLE
Set up redirects on nonexistent pages

### PR DESCRIPTION
**Pull request by sweatC**
Create posts in jtway about nonexistent pages.
Set up redirects on nonexistent pages.
photo_2018-05-20_11-53-09

### CODE REVIEW

**Comment by sweatC** [comment]
@pftg @obliviusm @andriyParashchuk pls check
Staging: http://jt-pr316.surge.sh/
Google Search Console
image

**Comment by andriyParashchuk** [comment]
@sweatC Do you need to redirect internal pages to a blog or homepage?
Because now it's redirected to the blog

**Comment by pftg** [comment]
@sweatC 

```
- /contact
+  - /career
+  - /credits
+  - /portfolio
+  - /portfolio/mostech
+  - /portfolio/oauth-to-me
+  - /portfolio/coach-alba
+  - /portfolio/greeting-pictures
```

need redirect to https://www.jetthoughts.com/

**Comment by pftg** [comment]
blog/team/2017/03/20/the-4-steps-to-bring-life-to-a-project -> should be redirected to post on jtway

**Review by pftg — CHANGES_REQUESTED** [review]
Please check comments

**Comment by pftg** [comment]
@sweatC please create issue to migrate posts from old blog to jtway if there is usage on GA for last 6 month

File: blog.html [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+  - /credits
+  - /portfolio
+  - /portfolio/mostech
+  - /portfolio/oauth-to-me
+  - /portfolio/coach-alba
+  - /portfolio/greeting-pictures
+  - /blog/index
+  - /blog/tech/2010/02/14/convert-bplist-to-xml
+  - /blog/tricks/2009/10/03/convert-authors-names-in-mercurial
+  - /blog/team/2017/03/20/the-4-steps-to-bring-life-to-a-project
```
Comment (pftg) [inline blog.html]: Should be redirected to https://jtway.co/the-4-steps-to-bringing-life-to-a-struggling-project-37a53bf9c7dc

Comment (sweatC) [reply blog.html]: Accepted

**Comment by sweatC** [comment]
@pftg issue created https://github.com/jetthoughts/jetthoughts.github.io/issues/323.
Redirects fixed.

File: blog.html [hunk 2]
```diff
@@ -1,5 +1,16 @@
 ---
 redirect_from:
+  - /contact
+  - /career
+  - /credits
+  - /portfolio
+  - /portfolio/mostech
+  - /portfolio/oauth-to-me
+  - /portfolio/coach-alba
+  - /portfolio/greeting-pictures
```
Comment (pftg) [inline blog.html]: please move them to home.html

**Review by pftg — CHANGES_REQUESTED** [review]
check comments

File: blog.html (hunk 2 above)
Comment (sweatC) [reply blog.html]: Accepted

**Comment by pftg** [comment]
@sweatC have you added issue to migrate some posts to jtway?

**Comment by pftg** [comment]
@sweatC please squash commits and ask for merge

**Comment by sweatC** [comment]
@pftg yes I have added https://github.com/jetthoughts/jetthoughts.github.io/issues/323. Check comment above.
It is ready for merge.

**Review by pftg — APPROVED** [review]
LGTM

**Pull request merged**